### DISCUSSION TITLE
Getting 0 data for 2 streams from Amazon Ads connection

### DISCUSSION THREAD

**Discussion by albert-le-whoop** [airbytehq/airbyte#39102] [Connector Questions]
I'm using Airbyte to query Amazon Ads API, and there's 0 data for `SPONSORED_BRANDS_REPORT_STREAM_METRIC` and `SPONSORED_BRANDS_REPORT_STREAM`. I've re-synced it twice, both with trying incremental dedupe approach and then full refresh overwrite, but there's still 0 data in the end tables (the other Snowflake tables get populated with data).

I can see non-video brand ad campaigns in `sponsored_brands_campaigns` table but the 2 above tables are empty which means I don't get any click or impressions data for those campaigns.

Anyone have this issue before?

**Comment by waltzforvenus** [comment]
Hey, having the same issue. sponsored_brands_report is not fetching, everything else is fine. Bumping in case anyone else also ran into this issue.

[a comment by an automated account is not shown]

**Reply by drewg2009** [reply]
I'm not getting any data on any streams for my Amazon Ads source account. 

When I login to Amazon Ads I can see all of the linked accounts. The source test and sync passes in Airbyte, but 0 profiles come back.
